hack/lib/util.sh

```diff
@@ -719,8 +719,11 @@ function kube::util::ensure-cfssl {
     cfssldir="${KUBE_TEMP}/cfssl"
   fi
 
-  mkdir -p "${cfssldir}"
-  pushd "${cfssldir}" > /dev/null
+  host_arch=$(kube::util::host_arch)
```

Comment:
suggestion for something slightly simpler than the change below, if we put something like this we don't need to make the larger if block below:
```bash
host_arch=$(kube::util::host_arch)
if [ "${host_arch}" != "amd64" ]; then
   echo "Cannot download cfssl on non-amd64 hosts and cfssl does not appear to be installed."
   echo "Please install cfssl and cfssljson and verify they are in \$PATH."
   echo "echo "Hint: export PATH=\$PATH:\$GOPATH/bin; go get -u github.com/cloudflare/cfssl/cmd/..."
fi

Reply:
Done.